themes/dark.json

```diff
@@ -107,11 +107,12 @@
     "editor.findMatchHighlightBackground": "#ffd33d22",
     "editor.inactiveSelectionBackground": "#3392FF22",
     "editor.selectionBackground": "#3392FF44",
-    "editor.selectionHighlightBackground": "#17E5E633",
-    "editor.wordHighlightBackground": "#17E5E600",
-    "editor.wordHighlightStrongBackground": "#17E5E600",
-    "editor.wordHighlightBorder": "#17E5E633",
-    "editor.wordHighlightStrongBorder": "#17E5E666",
+    "editor.selectionHighlightBackground": "#17E5E600",
+    "editor.selectionHighlightBorder": "#17E5E644",
+    "editor.wordHighlightBackground": "#17E5E633",
+    "editor.wordHighlightStrongBackground": "#17E5E622",
```

Comment:
I intentionally put less focus on the `strong` word highlight the regular one for those reasons:

* It's good to distinguish between the word where my cursor is and all the other occurrences to make it easier to navigate through the file.
* However, we need to distinguish between a highlight and a selection, so the contrast between those needs to be sufficient as well. Making the `strong` highlight more transparent helps a lot here.
* When we're searching for occurrences, we're not so much interested in the `strong` one.

Problems this could make:

* Distracts from the place where we placed our cursor. I hope that won't be too much of an issue as the currently selected line gets its own highlight.